source/extensions/common/wasm/wasm.cc

```diff
@@ -1068,11 +1139,20 @@ WasmResult serializeValue(Filters::Common::Expr::CelValue value, std::string* re
     result->assign(reinterpret_cast<const char*>(&out), sizeof(absl::Time));
     return WasmResult::Ok;
   }
+  case CelValue::Type::kMap:
+  case CelValue::Type::kList: {
+    ProtobufWkt::Value out;
+    if (!exportValue(value, &out)) {
+      return WasmResult::SerializationFailure;
```

Comment:
Is this the correct error?  What causes exportValue to fail?

Reply:
I think so. For now, it's enums and errors. I am not sure what the right export value for errors should be.